unit_tests/test_events.py

```diff
@@ -338,3 +342,28 @@ def test_failed_stall_during_filter(self):
                                line="[99.80.124.204] [stdout] Mar 31 09:08:10 warning|  reactor stall 20").publish()
 
         self.assertEqual(event.severity, Severity.DEBUG)
+
+    @pytest.mark.parametrize("test_input,expected", [(None, ''), (2, '2s'), (2.52, '2.52s'), (326, 326),
```

Comment:
Following looks as mistake:
```
(326, 326)
```

Shouldn't it be the following:
```
(326, "5m26s")
```
?
Does it really work as of now?

Reply:
All tests were passed. When I moved `@staticmethod` above on `@pytest.mark.parametrize`, it failed with error:\
```
TypeError: test_duration_format() missing 2 required positional arguments: 'test_input' and 'expected'
```

It's the bug in pytest that was fixed in 6.3.0.dev0:
https://github.com/pytest-dev/pytest/pull/5483/commits/95714436a1ec29e01f017c914cee63970acdcb2a

So we can't to use `@pytest.mark.parametrize` now.

Reply:
1) 2 years have passed since that bugfix.
2) What happens when you move "@staticmethod" below "@pytest.mark.parametrize" ?
3) Why do you need to use "@staticmethod"?

Reply:
> 1. 2 years have passed since that bugfix.

I know. But it was committed into 6.3.0.dev0
Do you have an idea? Another reason, why it fails?

> 2. What happens when you move "@staticmethod" below "@pytest.mark.parametrize" ?

When "@staticmethod"  is below "@pytest.mark.parametrize", test was marked as passed, but not really run.
 
> 3. Why do you need to use "@staticmethod"?

Because I got message, that "self" parameter is not expected

Reply:
Ok, pytest's "mark.parameterize" doesn't work with classes. Let's ignore that one.
You can use another already existing dependency in our repo -> parameterized.

So, what I asked for can be written in the following:

```
from parameterized import parameterized

...

class SctEventsTests(BaseEventsTest)

    ...

    @parameterized.expand([                                                                            
        (None, ''),                                                                                    
        (2, '2s'),                                                                                     
        (2.52, '2.52s'),                                                                               
        (326, '5m26s'),                                                                                
        (4598, '1h16m38s'),                                                                            
        (87400, '1d0h16m40s'),                                                                         
    ])                                                                                                 
    def test_duration_format_vp(self, duration_input, duration_formatted):                             
        event = NodetoolEvent(                                                                         
            nodetool_command="scrub --skip-corrupted drop_table_during_repair_ks_0",                   
            node='1.0.0.121', options="more options", publish_event=False)                             
                                                                                                       
        event.duration = duration_input                                                                
        self.assertEqual(event.duration_formatted, duration_formatted)
```

Reply:
Done